services/ui-src/src/components/modals/DeleteEntityModal.tsx

```diff
@@ -0,0 +1,65 @@
+// components
+import { Text } from "@chakra-ui/react";
+import { Modal, ReportContext } from "components";
+import { useContext, useState } from "react";
+// types
+import { AnyObject, EntityShape, ReportStatus } from "types";
+import { useUser } from "utils";
+
+export const DeleteEntityModal = ({
```

Comment:
This feels so similar to DeleteDynamicFieldRecordModal that I wonder if they could be combined. Like pass in the desired function ala`removeRecord`, pass in the desired text

Reply:
Hmm... they are very similar. I'm thinking for now let's keep them separate and review after we finalize the ModalDrawerReportPage changes through access measures, quality measures, and sanctions. There may yet be changes we're not anticipating.